src/com/rcv/Main.java

```diff
@@ -45,53 +45,61 @@ public static void main(String[] args) {
       String configPath = args[0];
       // config file for running the tabulator
       ElectionConfig config = loadElectionConfig(configPath);
-      Logger.info("Tabulator is being used via the CLI.");
-      executeTabulation(config);
+      if (config != null) {
+        Logger.info("Tabulator is being used via the CLI.");
+        executeTabulation(config);
+      } else {
+        Logger.severe("Aborting because config is invalid.");
+      }
     }
   }
 
   // function: loadElectionConfig
   // purpose: create config object
   // param: path to config file
-  // returns: the new ElectionConfig object or null if there was a problem
+  // returns: the new ElectionConfig object, or null if there was a problem
   static ElectionConfig loadElectionConfig(String configPath) {
     // config: the new object
     ElectionConfig config = null;
 
-    // encounteredError tracks whether anything went wrong during loading
-    boolean encounteredError = false;
+    // tracks whether we encountered any critical file system errors
+    boolean encounteredFileError = false;
```

Comment:
This variable is only set to true one time below, and it could just as easily be eliminated entirely and have `config = null;` replace the line where it's set to true. I'd recommend either doing that or else inserting `encounteredFileError = true;` in the other place where `config = null;` occurs, and also in the check for `rawConfig == null`. I think it's simpler just to get rid of it though.

Reply:
I actually need it, because when I encounter a file error, I don't want to set `config = null` immediately; first I want to call `config.getValidationErrors()`.